### ISSUE TITLE
Malformed path when using PathMerge with Path objects

### ISSUE THREAD

**Issue by 0x2b3bfa0** [joemaddalone/react-svg-path#69]
It looks like using `PathMerge` with `Path` object is not supported and produces a malformed output:

#### Input
```xml
<PathMerge>
    <PathOne />
    <PathTwo />
</PathMerge>
```

#### Actual output
```xml
<path d="<path d=&quot;M201.337···&quot; merge></path><path d=&quot;M30.4869 ···&quot; merge></path>" />
```

#### Expected output
```xml
<path d="M201.337··· M30.4869 ···" />
```

Do you have plans to support this kind of operation?

**Comment by joemaddalone** [comment]
@0x2b3bfa0, more than happy to give this a look and see if it can be resolved.  Can you provide a more detailed example of how to reproduce?  Are PathOne & PathTwo the product of `path.toComponent()` ?

**Comment by joemaddalone** [comment]
ah - I think I have repo'd correctly
```
const PathOne = () => <path d="M0,0h100v100" stroke="red" />
const PathTwo = () => <path d="M0,0v100h100" stroke="green" />
const App = () => {
  return (
      <Svg width={500} height={500}>
        <PathMerge>
          <PathOne />
          <PathTwo />
        </PathMerge>
      </Svg>
  );
};
```

digging into it now.

**Comment by 0x2b3bfa0** [comment]
@joemaddalone, sorry for the lack of a proper code example; you were quicker than me. 🏎️

**Comment by joemaddalone** [comment]
This is now fixed in v1.10.1

**Comment by 0x2b3bfa0** [comment]
🙏  Thank you very much, @joemaddalone!
